suite/runsuite_wrapper.pl

```diff
@@ -463,13 +463,14 @@
                                    'code_api|sample.signal' => 1); # i#3127
```

Comment:
> We do have a denylist for tests failing on AMD-32 in i#6417, which causes tests to not run at all. IMO, it is better to 

s/i#6417/#6417/ to get an embedded link

Reply:
Had intentionally prefixed the i#; there's already an embedded link just above this.